packages/block-editor/src/components/inspector-controls/groups.js

```diff
@@ -14,12 +14,17 @@ const InspectorControlsTypography = createSlotFill(
 	'InspectorControlsTypography'
 );
 
+// The general group is intended to render fills relevant to a block as a whole.
+// As such, they will normally be rendered outside the block inspector's tabs.
+const InspectorControlsGeneral = createSlotFill( 'InspectorControlsGeneral' );
```

Comment:
Very open to better names for this inspector controls group. Some of the others considered included: `highlighted`, `promoted`, `important`, `block-wide`, etc.

Reply:
What's the point of this one? I don't think we should support this slot if it means what I think it means :)

Reply:
This was the slot [touched on earlier](https://github.com/WordPress/gutenberg/pull/45005#issuecomment-1291762267) to allow items that apply to the block as a whole to be rendered above the sidebar tabs. Perhaps a better name for the inspector controls group would be `BlockInformation`?

I can see it potentially being abused by some third-party devs to get their plugin's controls a more prominent position. I've moved this slot to a separate PR ( https://github.com/WordPress/gutenberg/pull/45437 ) to discuss the best course of action further.

In the meantime, we might be able to land this PR and begin the experiment and get wider feedback. Are you happy with that plan?